plugins/arDominionPlugin/css/less/forms.less

```diff
@@ -355,6 +355,25 @@ button.delete-small {
   content: '.';
 }
 
+// Rename model form-item div prevents clicks to enable/disable checkboxes
+#renameModalForm {
+  div.modal-body {
+    div.form-item {
+      display: inline;
+    }
+  }
+}
+
+#renameModalForm {
+  div.modal-body {
+    div.form-item {
+      label {
+        display: inline;
+      }
+    }
+  }
+}
+
```

Comment:
I'm not sure why you need both elements to be inline, but just focusing on the syntax, could you just convert that snippet into something equivalent like the following?

```css
#renameModalForm .modal-body {
  .form-item, label {
    display: inline;
  }
}
```